languages/ruby/exercises/concept/exceptions/.meta/example.rb

```diff
@@ -0,0 +1,16 @@
+class SimpleCalculator
+  ALLOWED_OPERATIONS = ['+', '-', '/', '*']
+
+  UnsupportedOperation = Class.new(StandardError)
+
+  def self.calculate(first_operand, second_operand, operation)
+    raise ArgumentError.new unless (first_operand.is_a?(Integer) && second_operand.is_a?(Integer))
```

Comment:
These should be valid for all numerics, not only `Integer`, the division being the guarded operation for division by zero.  Even if not guarding against 0, but for "Type" instead, `Integer` is likely too specific.

Reply:
It makes sense, I'll change to `Numeric`